fatiando/mesher.py

```diff
@@ -1720,3 +1720,433 @@ def keep(cell):
             return True
         return False
     return [c for c in cells if keep(c)]
+
+
+class TriaxialEllipsoid (GeometricElement):
+
+    """
+    Create a triaxial ellipsoid.
+
+    .. note:: The coordinate system used is x -> North, y -> East
+    and z -> Down
+
+    Parameters:
+
+    * x, y, z : float
+        The coordinates of the center of the ellipsoid
+    * a, b, c : float
+        Semi-axes of the ellipsoid (a>b>c)
+    * alpha, gamma, delta
+        Orientation angles of the ellipsoid
+    * props : dict
+        Physical properties assigned to the ellipsoid.
+        Ex: ``props={'density':10,
+                     'remanence':[10000., 25., 40.],
+                     'k':[0.562, 0.485, 0.25, 90., 0., 0.]}``
+
+    Examples:
+
+        >>> e = TriaxialEllipsoid(1, 2, 3, 6, 5, 4, 10, 20, 30,{
```

Comment:
When creating the instance, use the argument names to make it easier to understand. There is no way that I can know what `1, 2, 3, 6, 5, 4, 10, 20, 30` means for the ellipsoid. Use `TriaxialEllipsoid(x=1,0 y=2, z=3, a=6, b=5, c=4, alpha=10, beta=20, gamma=30, props={...`.